vars/buildKibanaDockerImage.groovy

```diff
@@ -85,16 +86,23 @@ def call(Map args = [:]){
   log(level: 'DEBUG', text: "Tagging ${dockerImageSource}:${kibanaVersion} to ${dockerImageTarget}:${kibanaDockerTargetTag} and ${dockerImageTarget}:${deployName}")
 
   retryWithSleep(retries: 3) {
-    sh(label: 'Push Docker image', script: """
-      docker tag ${dockerImageSource}:${kibanaVersion} ${dockerImageTarget}:${kibanaDockerTargetTag}
-      docker tag ${dockerImageSource}:${kibanaVersion} ${dockerImageTarget}:${deployName}
-      docker tag ${dockerImageSource}:${kibanaVersion} ${dockerImageTarget}:${kibanaVersion}-${kibanaDockerTargetTag}
-      docker tag ${dockerImageSource}:${kibanaVersion} ${dockerImageTarget}:${kibanaVersion}-${deployName}
-      docker push ${dockerImageTarget}:${kibanaDockerTargetTag}
-      docker push ${dockerImageTarget}:${deployName}
-      docker push ${dockerImageTarget}:${kibanaVersion}-${kibanaDockerTargetTag}
-      docker push ${dockerImageTarget}:${kibanaVersion}-${deployName}
-    """)
+    def tags = [
+      "${kibanaDockerTargetTag}",
+      "${deployName}",
+      "${kibanaVersion}-${kibanaDockerTargetTag}",
+      "${kibanaVersion}-${deployName}
+    "]
+    def dockerImages = ["${dockerImageSource}", "${dockerCloudImageSource}"]
+    tags.each{ tag ->
+      dockerImages.each { dockerImage ->
+        def src = "${dockerImage}:${kibanaVersion}"
+        def dst = "${dockerImage}:${tag}"
+        sh(label: 'Push Docker image', script: """
+          docker tag ${src} ${dst}
```

Comment:
NIT: it's possible to pass multiple tags to one single `docker tag` command passing multiple `-t TAG1 -t TAG2` flags

Reply:
I know, but the resulting logic to compose a string like `docker ${src} -t TAG1 -t TAG2 ...` will result odd, more complex to follow, and do not improve the build speed in a sensible time.